### PR TITLE
Fixed Employee ID Missing on new users

### PR DESCRIPTION
**Pull request by gillespr-truvant**
Added Line in new-domainuser function to include employeeID from the user attribute prompt.

### CODE REVIEW

File: NDU-Functions.psm1 [hunk 1]
```diff
@@ -59,7 +59,7 @@ Load the email template from the HTML file and replaces the placeholders with us
     
 	Write-PSFMessage -Level Host -Message "Attempting to send email to $($manager) for the user $($user) with password: $($Emailpassword)"
     #Get the mail template
-    $mailTemplate = (Get-Content ("$PSScriptRoot\Resources\MailTemplateUserCreated.html")) | ForEach-Object {
```
Comment (rdglabs) [inline NDU-Functions.psm1]: Leave with $PSScriptRoot

File: NDU-Functions.psm1 [hunk 2]
```diff
@@ -313,7 +314,7 @@ Generates a passphase with two words from the password dict. Adjust the words to
 #>
     
 
-    $Dict = Get-Content -Path "$PSScriptRoot\Resources\password.dict" | Sort-Object {Get-Random} | Select-Object -First $config.Settings.password.NumberofWords
```
Comment (rdglabs) [inline NDU-Functions.psm1]: Leave with $PSScriptRoot

File: NDU-Functions.psm1 [hunk 3]
```diff
@@ -340,7 +341,7 @@ Generates a passphase with two words from the password dict. Adjust the words to
     if($config.Settings.password.Numbers){
         $Delimiters += '1','2','3','4','5','6','7','8','9','0'   
     }
-    if($config.Settings.password.SpecialChar.count -gt 0){
```
Comment (rdglabs) [inline NDU-Functions.psm1]: Change back

File: NDU-Functions.psm1 [hunk 4]
```diff
@@ -370,7 +371,7 @@ Generates a passphase with two words from the password dict. Adjust the words to
 
     
     Write-PSFMessage -Level Host -Message "New Password generated: $($global:Password)`n`n" 
-	pause
```
Comment (rdglabs) [inline NDU-Functions.psm1:373]: Pause it better to let user copy the password

File: NDU-Functions.psm1 [hunk 5]
```diff
@@ -402,7 +403,7 @@ Creates a schedule task that will executes the scheduleEntraGroupCopy.ps1 script
     $user = $cred.UserName
     $pass = $cred.GetNetworkCredential().Password
     $trigger = New-ScheduledTaskTrigger -Once -At $taskDate
-    $action  = New-ScheduledTaskAction -Execute "C:\Program Files\PowerShell\7\pwsh.exe" -Argument '-ExecutionPolicy Bypass -file "$PSScriptRoot\ScheduleEntraGroupCopy.ps1"' -WorkingDirectory "$PSScriptRoot"
```
Comment (rdglabs) [inline NDU-Functions.psm1:405]: Leave with $PSScriptRoot

File: NDU-Functions.psm1 [hunk 6]
```diff
@@ -448,7 +449,7 @@ adds the targetuser info and sourceuser to a csv file that is used by ScheduleEn
         Completed = $EntraCompleted
     }
     try {
-        $csv | export-csv $PSScriptRoot\Resources\EntraUsers.csv -Force -Append
```
Comment (rdglabs) [inline NDU-Functions.psm1]: Leave with $PSScriptRoot

File: NDU-Functions.psm1 [hunk 7]
```diff
@@ -858,9 +859,9 @@ function Get-DomainUserData {
         $attributes.add("office",$config.site.$($office.Code).office)
     }
     #OU
-    ##$attributes.add("OU",$config.site.$($office.Code).OU)
+    #$attributes.add("OU",$config.site.$($office.Code).OU)
     try {
-        if(Get-ADOrganizationalUnit -Identity $($config.site.$($office.Code).OU) -ErrorAction stop) {
+        if(Get-ADOrganizationalUnit -Identity $($config.site.$($office.Code).OU) -erroraction stop) {
```
Comment (rdglabs) [inline NDU-Functions.psm1]: correct caps

**Review by rdglabs — CHANGES_REQUESTED** [review]

**Review by gillespr-truvant — COMMENTED** [review]
Corrected Requests

**Review by rdglabs — DISMISSED** [review]

**Comment by gillespr-truvant** [comment]
Running a test

**Review by rdglabs — APPROVED** [review]

**Pull request merged**